api/src/test/socketioTestClient.ts

```diff
@@ -9,6 +9,7 @@ export type socketioTestRequest = {
     changeRequest?: ChangeReqDto;
     timeout?: number;
     getAccessMap?: boolean;
+    accessMap?: any; // Using 'any' here to accommodate varied structures for permissions data, which may differ based on client requirements or evolve over time.
```

Comment:
This sounds like AI...
```suggestion
    accessMap?: any; // The accessMap is converted to a AccessMap object by the API.
```